docs/how-to-monitor-minio-with-prometheus.md

```diff
@@ -10,7 +10,22 @@ This document explains how to setup Prometheus and configure it to scrape data f
 
 MinIO server release `RELEASE.2018-05-11T00-29-24Z` or later running. To get started with MinIO, refer [MinIO QuickStart Document](https://docs.min.io/docs/minio-quickstart-guide). Follow below steps to get started with MinIO monitoring using Prometheus.
 
-### 1. Download Prometheus
+### 1. Configure Prometheus in MinIO
+
+Prometheus is enabled and the authentication is set to "public" by default in MinIO configuration. For a secured access, update the prometheus config as follows
+
+``` json
+    "prometheus": {
+        "auth": "jwt",
+        "enable": true
+    }
+```
```

Comment:
Need more specific information on setting this via `mc admin config set`, or reference to config documents.

Reply:
Since we no more rely on the server config, we dont deal with config sets/gets